src/random.c

```diff
@@ -144,10 +144,10 @@ random_get_bytes (Random    *random,
     }
     return i;
 }
+
```

Comment:
whitespace: https://github.com/tpm2-software/tpm2-abrmd/blob/master/CONTRIBUTING.md#patch-requirements

Reply:
Removed, although it's unclear what the whitespace rule is here. Whitespace between functions makes it more readable.